usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -122,7 +118,7 @@ def account_download_filter(account_type, download_table, filters, account_level
         )
 
     # Apply filter and return
-    return queryset.filter(dod_filter, **query_filters)
```

Comment:
I assume this filter was removed because subsuming dod agencies is being done by the fed account bucketing now?

Reply:
DOD stuff should never ever be used anywhere ever again.  Ever.  Except in the agency/account linker.  But nowhere else.  Ever.